Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -97,47 +97,24 @@ private extension EmptyCountRule {
         }
 
         override func visit(_ node: InfixOperatorExprSyntax) -> ExprSyntax {
-            guard node.hasBinaryOperator else {
+            guard node.hasBinaryOperator, let binaryOperator = node.binaryOperator else {
```

Comment:
This reads strange now: Looks like the first condition isn't needed because `node.binaryOperator` is optional. I suggest something like:
```swift
guard let binaryOperator = node.binaryOperator, binaryOperator.isComparison else {
    // ...
}
```
This should be reused in line 80.

Reply:
Fixed at 415b1d7